### PR TITLE
Adds defibrillators, ported from Baystation.

### PR DESCRIPTION
**Pull request by NonQueueingMatt**
**DEFIBS DO NOT RESUSCITATE BRAINDEAD PEOPLE, THEY RESTART THE HEART ONLY!!!!!!**

Ported defibrillators from Bay while changing their sprites and functionality. Base defibs cannot be wielded and must be placed somewhere to be used - click and drag the defib to yourself to take out the paddles, then wield them. For compact defibrillators, simply equip them and then click them with an empty hand. To take them off, clickdrag to an empty hand.

CPR has been slowed down to compensate for these existing.

### CODE REVIEW

**Comment by NonQueueingMatt** [comment]
!review

**Comment by TheDanseMacabre** [comment]
Do they fix asystole?

File: code/game/objects/items/defib.dm [hunk 1]
```diff
@@ -0,0 +1,704 @@
...
+		return (R.cell && R.cell.checked_use(charge_amt))
+
+/obj/item/shockpaddles/rig
+	name = "mounted defibrillator"
+	desc = "If you can see this something is very wrong, report this bug."
+	cooldowntime = (4 SECONDS)
+	chargetime = (1 SECOND)
+	chargecost = 150
+	safety = 0
+	wielded = 1
```
Comment (Arrow768) [inline code/game/objects/items/defib.dm:569]: TRUE/FALSE

File: code/game/objects/items/defib.dm [hunk 2]
```diff
@@ -0,0 +1,704 @@
...
+		to_chat(user, SPAN_WARNING("They are missing that body part!"))
+		return
+
+	//no need to spend time carefully placing the paddles, we're just trying to shock them
+	user.visible_message(SPAN_DANGER("\The [user] slaps [src] onto [H]'s [affecting.name]."), SPAN_DANGER("You overcharge [src] and slap them onto [H]'s [affecting.name]."))
+
+	//Just stop at awkwardly slapping electrodes on people if the safety is enabled
+	if(safety)
+		to_chat(user, SPAN_WARNING("You can't do that while the safety is enabled."))
+		return
```
Comment (Arrow768) [inline code/game/objects/items/defib.dm:451]: shouldnt check be performed before displaying a visible message about overcharging the defi?

Comment (NonQueueingMatt) [reply code/game/objects/items/defib.dm:451]: intentional as per the comment

**Review by Arrow768 — APPROVED** [review]

**Review by Alberyk — APPROVED** [review]

**Pull request merged**